hq/app/model/models.scala

```diff
@@ -333,16 +333,21 @@ case class VulnerableAccessKey(
   humanUser: Boolean
 )
 
-sealed trait IamAuditNotificationType {def name: String}
-object Warning extends IamAuditNotificationType {val name = "Warning"}
-object Final extends IamAuditNotificationType {val name = "Final"}
```

Comment:
I hadn't realised that these `Warning` and `Final` objects were unused, so good to get rid of them!